controllers/accounts.ts

```diff
@@ -27,21 +27,36 @@ import { urlFriendly } from "../library/format";
 import { upload_image } from "../library/artwork";
 import { getZBDRedirectInfo, getZBDUserInfo } from "../library/zbd/login";
 import { updateNpubMetadata } from "../library/nostr/nostr";
+import {
+  uniqueNamesGenerator,
+  adjectives,
+  colors,
+  animals,
+} from "unique-names-generator";
+
+function makeRandomName() {
+  return uniqueNamesGenerator({
+    dictionaries: [adjectives, colors, animals],
+    separator: "-", // word separator
+  }); // example: big-red-donkey
+}
 
-async function groupSplitPayments(combinedAmps) {
-  // Group records by txId
-  const grouped = combinedAmps.reduce((acc, curr) => {
-    // User createdAt as identifier for legacy amps
-    const identifier = curr.txId ? curr.txId : curr.createdAt;
-    if (!acc[identifier]) {
-      acc[identifier] = [];
-    }
-    acc[identifier].push(curr);
-    return acc;
-  }, {});
-
-  // convert grouped to array
-  return Object.keys(grouped).map((key) => grouped[key]);
+async function checkName(name): Promise<string> {
+  if (name && name.length > 0) {
+    return name;
+  }
```

Comment:
may be a good idea to remove leading/trailing whitespace
```suggestion
  if (name && name.trim().length > 0) {
    return name.trim();
  }
```